app/src/main/java/com/pajato/android/gamechat/main/MainActivity.java

```diff
@@ -140,16 +136,7 @@ public void onClick(final View view) {
         // Handle navigation view item clicks here by posting a click event and closing the drawer.
         String format =  "Navigation Item Selected on view: {%s}";
         Log.v(TAG, String.format(Locale.US, format, item.getClass().getSimpleName()));
-        switch (item.getItemId()) {
```

Comment:
Why did you delete this? It stops all the functionality that I just put into the navigation drawer.